clang/lib/AST/Interp/IntegralAP.h

```diff
@@ -100,12 +100,13 @@ template <bool Signed> class IntegralAP final {
   }
   static IntegralAP from(const Boolean &B) {
     assert(false);
-    return IntegralAP::zero();
+    return IntegralAP::zero(1);
```

Comment:
Why does this not look at the value of `B`?

Reply:
This is unimplemented right now, I just added the bitwidth to make it compile.

Reply:
Ah, please add FIXME comments to unimplemented stuff when you can.

(I'll be honest -- reviewing new interpreter patches is getting really difficult because of the interdependence between so many unrelated reviews and how much stuff is half-done. That's why these reviews keep getting delayed on my end, there's just too much to try to keep track of spread out across too many reviews. I wish we could find a way to improve this without impeding your ability to make progress. CC @Shafik @erichkeane @cor3ntin as other folks doing a fair number of these reviews in case this is a shared concern and folks have ideas on how to improve it.)

Reply:
I don't have such comments because I opted for `assert(false)` where things are unimplemented :)

I keep opening smaller PRs _to make review easier_ because I thought that... makes it easier. If larger (but complete) patches are better for you, I can try to do that instead.

Reply:
Removed that code in https://github.com/llvm/llvm-project/commit/66c99154a130553a50e499d898ef1fba5b755dcf.